index.d.ts

```diff
@@ -24,8 +24,15 @@ declare namespace authService {
   function setFormsKeyToken(formsKeyToken: string): void
   function isAuthorised(formsAppId: number): Promise<boolean>
   function requestAccess(formsAppId: number): Promise<void>
-  function login(): Promise<string>
+  function loginHostedUI(identityProvider: string | void): Promise<string>
```

Comment:
```suggestion
  function loginHostedUI(identityProvide?r: string): Promise<string>
```

Reply:
Actually, can we use an enum of the possible options?

Reply:
No we can't SAML identity providers are custom strings.